### DISCUSSION TITLE
Import from Mela

### DISCUSSION THREAD

**Discussion by wbrawner** [mealie-recipes/mealie#1995] [Feature Request]
**Before submitting this feature request I have**

- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
[Mela](https://mela.recipes/) is a native recipe management app exclusively available for Apple platforms. It allows users to export their data but uses [its own file format](https://mela.recipes/fileformat/index.html). Importing this data thus has to be done manually. There are already other import solutions so it shouldn't be too much work to add an implementation for Mela.

**Additional Information**

- [x] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Always happy to accept a PR for a new import type!

**Comment by Ntgllr** [comment]
If anyone searches for a way to migrate from Mela to Mealie: I hacked together a small Python tool to import Melas recipe files: [Ntgllr/migrate-mela-mealie](https://github.com/Ntgllr/migrate-mela-mealie)

Full disclosure: I'm not very good at coding, so I used ChatGPT Codex to create the code. There may be some bugs, but it worked for me.

A PR for the import function would be a lot cleaner and preferrable, of course. If I find the time I will take a shot at it (No promises I will finish it, though).
